tsschecker/tsschecker.c

```diff
@@ -519,19 +531,10 @@ t_versionURL *getFirmwareUrls(const char *deviceModel, t_iosVersion *versVals, j
     return (t_versionURL*)rets_base;
 }
 
-static void printline(int percent){
-    info("%03d [",percent);for (int i=0; i<100; i++) putchar((percent >0) ? ((--percent > 0) ? '=' : '>') : ' ');
-    info("]");
-}
-
-static void fragmentzip_callback(unsigned int progress){
-    info("\x1b[A\033[J"); //clear 2 lines
-    printline((int)progress);
-    info("\n");
-}
+static void fragmentzip_callback(){}
```

Comment:
undo removal of this callback, it works on most devices, doesn't work on some, so disable it on the os it fails on instead of removing it for all os's